butte/src/types.rs

```diff
@@ -0,0 +1,334 @@
+//! Types representing the parts of a flatbuffer schema
+use derive_more::From;
+use std::collections::HashMap;
+use std::path::Path;
+use typed_builder::TypedBuilder;
+
+/// A flatbuffer schema
+#[derive(Debug, Clone, PartialEq, TypedBuilder)]
+pub(crate) struct Schema<'a> {
```

Comment:
It would be great to export the flatbuffers IDL Schema so that other crates can reuse this AST.  I understand it can be useful as an implementation details, but until 1.0 there's no need to keep backwards compatibility.

Reply:
Everything in `types.rs` is now `pub`.